tasks/rslang/english-for-kids-translated.md

```diff
@@ -0,0 +1,153 @@
+- when you click on the card, you will hear the the pronunciation of the word in English
+- each card has a button on the right bottom corner, when you click on that button the card flips over. The back side of the card has a translation of the word. Clicking on the back side of the card does not trigger any events like pronunciation of the word.
+- card is rotated back to the front side when the mouse cursor moves beyond the borders of the card
+
+3. Statistics Page
+- the page description is added in the evaluation criteria (Hacker scope)
+
+## How Application should work
+The application works in training and game modes.
+Description of the application in these modes is added in evaluation criteria (Basic scope and Advanced scope, respectively). When you download the application or reload the page, the application opens in training mode. Switching between training and playing modes is triggered by toggling the Train / Play switch button.
+
+## Repository requirements
+- to develop the application use school’s private repository. [How to work with a private repository](https://docs.rs.school/#/stage2?id=Как-работать-с-приватным-репозиторием)
+- the commit history should reflect the application development process. [Commit Requirements](https://docs.rs.school/#/git-convention)
+- the demo version of the application should be available at  `https://www.netlify.com/`, or on another similar hosting. For demos hosted on netlify.com, the name of the site should in the following format: name of the github account - name of the task.
+- after the end of development process or just before the deadline, create a pull request from the english-for-kids branch to the master branch. [Pull request requirements.](https://docs.rs.school/#/stage2?id=Описание-pull-request-должно-содержать-следующую-информацию). Do not merge the pull request.
+- application is checked by mentor and other students (cross-check)
+- a link to the demo version of the application has to be added to rs-app to let other student asses the application during the cross-check
+
+## Technical requirements
+- the application is checked in the latest version of Google Chrome
+- use of jQuery is not allowed
+- you are not allowed to use Angular / React / Vue
+- you are allowed to use bootstrap and/or any other css frameworks
+- you are allowed to use html and css preprocessors
+- you decide whether to use a single-page (SPA) or multi-page (MPA) application development approach, as well as the implementation of navigation between pages
+  - if you choose a single-page application (SPA) approach, you do not have to implement all the features of this approach (for example, routing) if they are not critical
+
+## Application Design Requirements
+- minimum width at which the application is displayed correctly is 320 px
+- since the application is intended for kids, including those who cannot read, all labels should be accompanied with pictures or icons if possible
```

Comment:
```suggestion
- since the application is intended for kids, including those who aren't able to read, all labels should be accompanied by pictures or icons if possible
```